tfjs-backend-webgpu/src/kernels/crop_and_resize_webgpu.ts

```diff
@@ -42,8 +41,8 @@ export class CropAndResizeProgram implements WebGPUProgram {
     this.dispatch = computeDispatch(
         this.dispatchLayout, this.outputShape, this.workGroupSize);
 
-    this.shaderKey =
-        `cropAndResize_${method}_${cropSize}_${extrapolationValue}`;
+    this.shaderKey = `cropAndResize_${method}_${cropSize}_${
+        extrapolationValue}_${imageShape}_${boxShape}_${this.outputShape}`;
```

Comment:
Why do you still need this since they are all in the uniforms?

Reply:
We need some extra efforts to remove this from shaderKey.

Reply:
There are so many const in this program, imageHeight.  I need more understand of this, suggest do this in another PR.